psqlgraph/psql.py

```diff
@@ -235,25 +263,22 @@ def _configure_driver_mappers(self):
                 'Have you imported your models?'
             ).format(str(e)))
 
-    def voided_nodes(self, query=VoidedNode):
+    def voided_nodes(self, query=None):
+        query = query or VoidedNode
         with self.session_scope(must_inherit=True) as local:
             if isinstance(query, list) or isinstance(query, tuple):
                 return local.query(*query)
             else:
                 return local.query(query)
 
     def voided_edges(self, query=VoidedEdge):
```

Comment:
Do we want to give this default parameter the same treatment as the `voided_nodes` method above?

Reply:
Yes that makes sense. Also, taking a closer look it now means one can do something like this 
``` python
self.voided_nodes(query=VoidedEdge)
```
It works but contradicts the name of the method. I'll try a refactor around this to make usage clearer